SpeechResponder/EditScriptWindow.xaml

```diff
@@ -10,6 +11,80 @@
         ResizeMode="CanResizeWithGrip"
         Height="600" Width="800">
     <DockPanel LastChildFill="True" Background="#FFE5E5E5">
+        <ToolBar DockPanel.Dock="Top" Background="#FFE5E5E5">
```

Comment:
The "grip" on the left side of the toolbar can be hidden by adding `ToolBarTray.IsLocked="True"`.

Reply:
Done